lib/auth.ts

```diff
@@ -520,18 +520,30 @@ const getAuth = function (
 		const email = verificationPayload.email;
 		const verificationToken = verificationPayload.token;
 
-		const { body } = await request.send({
-			method: 'POST',
-			url: '/user/v1/verify-email',
-			body: {
-				verificationToken,
-				email,
-			},
-			baseUrl: apiUrl,
-			sendToken: false,
-		});
+		try {
+			const { body } = await request.send({
+				method: 'POST',
+				url: '/user/v1/verify-email',
+				body: {
+					verificationToken,
+					email,
+				},
+				baseUrl: apiUrl,
+				sendToken: false,
+			});
 
-		return body;
+			return body;
+		} catch (err) {
+			if (err.statusCode === 400) {
```

Comment:
This could also be b/c of providing an invalid email or verification code and doesn't suffice to reason whether the provided code was expired. Maybe we should also match the returned error message as well and throw a `new BalenaError`.

Reply:
Do we want to segregate the type of error on the UI or update the wording of the above error msg to be a bit more generic? e.g. `'Issue with the verification link provided - log in again to generate a new one.'`

Reply:
Let's just go with "Your verification link has expired or is invalid - log in again to request another one."